### PR TITLE
yashim/hotfix: fix failed test due to directly importing sub-dependency

### PR DESCRIPTION
**Pull request by yashim-deriv**
Co-authored-by: Yashim Wong <yashimwong@gmail.com>

Changes:
- Import from `gatsby` instead of `@reach/router`

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/help-centre/index.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import { matchSorter } from 'match-sorter'
 import styled, { css } from 'styled-components'
 import { Helmet } from 'react-helmet'
-import { navigate } from '@reach/router'
+import { navigate } from 'gatsby'
```
Comment (mustofa-binary) [inline src/pages/help-centre/index.js:5]: does this navigation localized?

Comment (yashim-deriv) [reply src/pages/help-centre/index.js:5]: It's only used for navigating within the help-centre using anchor links.

Comment (sean-binary) [reply src/pages/help-centre/index.js:5]: The function that utilises `navigate` here is not actually being called by the component. We can omit it in the future.

Comment (yashim-deriv) [reply src/pages/help-centre/index.js:5]: The function that references `navigate` doesn't seem like is used anywhere. Should i remove this?

Screenshot 2021-03-19 at 11 31 54 AM

Comment (mustofa-binary) [reply src/pages/help-centre/index.js:5]: we can remove that in the future

**Review by sean-binary — APPROVED** [review]

**Pull request merged**